src/test/groovy/com/github/vkorobkov/jfixtures/integration/CleanTableIntegrationTest.groovy

```diff
@@ -0,0 +1,62 @@
+package com.github.vkorobkov.jfixtures.integration
+
+import com.github.vkorobkov.jfixtures.testutil.H2Test
+import spock.lang.Specification
+
+import java.nio.file.Path
+
+class CleanTableIntegrationTest extends Specification implements H2Test {
+    Path tmpCleanFolder = unpackYamlToTempFolder("clean_table.yml")
+
+    void setup() {
+        recreateTable("friends", "ID INT, NAME VARCHAR(50), AGE INT")
+        recreateTable("mates", "ID INT, NAME VARCHAR(50), AGE INT")
+        recreateTable("users", "ID INT, NAME VARCHAR(50), AGE INT")
+    }
+
+    def cleanup() {
+        tmpCleanFolder.toFile().deleteDir()
+    }
+
+    def "should delete from table by default"() {
+        given:
+        sql.execute("INSERT INTO mates (\"id\", \"name\", \"age\") VALUES (10000, 'Igor', 31);")
+        def result = sql.rows("SELECT * FROM mates")
```

Comment:
Let's remove these `result.size() == 1` checks in `given:` blocks.
If SQL didn't throw an exception means that the record was added. Also, the last test checks that `secondResult.size() == 2` which also means the record was added.